src/up.cmd.js

```diff
@@ -126,6 +137,28 @@ export default class UpCommand extends AbstractCommand {
       this._project.withCacheDirectory(this._cache);
     }
 
+    if (this._tls) {
+      if (
+        !this._tlsCertPath
+        || !this._tlsKeyPath
+      ) {
+        throw Error(chalk`{redIf using TLS, you must provide both tls cert and tls key...one or both not found`);
```

Comment:
typo `redIf`...

Reply:
Sorry, I got the formatting wrong.  I just pushed a fix to this.